src/commands/backlog/checkin.ts

```diff
@@ -6,32 +6,40 @@ import {AxiosBitBucketClient} from '../../api/service/axios-bitbucket-client'
 type DetailedIssue = Issue & {branch: any; pullRequest: any}
 
 export default class BacklogCheckin extends GitBaseCommand {
-  static description = 'Push local changes (commits) to the remote repository'
+  static description = 'After backlog:checkout and committing changes, push local changes to the remote repository and create/update a pull request.'
+
+  static examples = [
+    `# nrdk backlog:checkout <Jira issue ID>
+     # git add .
+     # git commit -m "Adding new feature"
```

Comment:
git commit -m "<Jira issue ID> Adding new feature" ?

Reply:
Works for me!